src/test/java/com/cloudbees/jenkins/support/slowrequest/InflightRequestTest.java

```diff
@@ -62,7 +62,7 @@ public HttpResponse doSubmitReferer(StaplerRequest req, StaplerResponse rsp) thr
                                  "    <a href='" + req.getContextPath() + "/mockSlowURLCall/submit"+ "' id='link'>link</a>" +
                                  "  </body>" +
                                  "</html>";
-            return HttpResponses.html(literalHTML);
+            return HttpResponses.literalHtml(literalHTML);
```

Comment:
It does `flush` whereas `html` doesn't, not sure if it may impact posterior management of the streams

Reply:
This is just a mock inside a test.